modules/enforce-lqlabel-none.adoc

```diff
@@ -6,7 +6,13 @@
 [role='_abstract']
 The local-queue labeling policy is enforced in all projects by default.
 
-To disable the policy for all projects, follow these steps.
+You can disable the policy for all projects in either of the following ways:
+
+* Delete the `kueue-validating-admission-policy-binding` resource.
+The resource is automatically re-created with the default values. 
+No other action is required.
```

Comment:
Deleting this resource will re-create it with the default values, meaning, the policy will be enforced onto all projects. As this is the default behaviour.

This part could probably go in the `enforce-lqlabel-all` doc.

Reply:
Great catch, @ChristianZaccaria, thank you! I've moved that text to the correct module now, apologies for the error!